### ISSUE TITLE
Focus rings are not visible in the light version

### ISSUE THREAD

**Issue by Kiwow** [Kiwow/symbol_website#3]

**Comment by Kiwow** [comment]
This was fixed by 4cc23aafceb24ea803be97a26627e4c4b53ca510
